Firmata.cpp

```diff
@@ -66,7 +80,8 @@ FirmataClass::FirmataClass()
   firmwareVersionCount = 0;
   firmwareVersionVector = 0;
   blinkVersionDisabled = false;
-  systemReset();
```

Comment:
I should also mention the FirmataParser constructor will be implicitly called during the FirmataClass constructor.